docs/reference/islandora_defaults_reference.md

```diff
@@ -0,0 +1,395 @@
+| media         | audio               | Do not hide untranslatable fields.          |
+| media         | file                | Do not hide untranslatable fields.          |
+| media         | image               | Do not hide untranslatable fields.          |
+| media         | video               | Do not hide untranslatable fields.          |
+| taxonomy_term | islandora_access    | Language alterable is false.         |
+| taxonomy_term | islandora_media_use | Language alterable is false.         |
+| taxonomy_term | islandora_models    | Language alterable is false.         |
+
+## Views
+
+(see Glossary: [View](../user-documentation/glossary.md#view))
+
+| Name                                                | Description                                    | Displays                                                                                     | Comments                                               |                                                                                                                                  |
+|-----------------------------------------------------|------------------------------------------------|----------------------------------------------------------------------------------------------|--------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| IIIF Manifest (iiif_manifest)                       | Generates IIIF manifests for paged content     | REST Export (/node/%node/book-manifest);<br/>REST Export - Single node (/node/%node/manifest) |                                                        |                                                                                                                                  |
+| Members (members)                                   | Displays members for content.                  | Members (block)                                                                              | Show an unformatted list of published nodes that are "Member of" the current node (from URL), formatted as "Teaser"s, paged to 10 items.                                                      |                                                                                                                                  |
+| OpenSeadragon Media EVAs (openseadragon_media_evas) | Displays media for content as EVA's per model. | Original File;<br/>Preservation Master;<br/>Service File | Show 1 Media that is "Media of" the node (from URL) and having Media Use corresponding to each Display (Original file, etc.). The Media is rendered with display mode OpenSeadragon. See media.\*.open_seadragon under "Display Modes." The description's "per model" should read "per media use". Provides the EVA used in the islandora_object.open_seadragon display mode. The Preservation Master and Service File displays are not used. |
```

Comment:
That comment's a mouthful.  Maybe say "Provides fields to show files using the Openseadragon viewer. Each Media Use gets its own field, so the user can choose between displaying the Original File or a derivative such as the Service File."?